### PR TITLE
Move 'expect' to devDependencies in package.json

### PR DESCRIPTION
**Pull request by charakajg**
**Issue**
The "expect" library is only used in test files. However, in package.json, it has incorrectly listed as a dependency instead of as dev dependancy. Therefore, whenever someone who uses this hellosign-nodejs-sdk library runs "npm audit --production", it incorrectly shows any issues with "expect" library, although it's not used in production.

**Fix**
Here, the "expect" library has been moved to its rightful place "devDependencies" in package.json

### CODE REVIEW

**Comment by gabegorelick** [comment]
https://github.com/HelloFax/hellosign-nodejs-sdk/issues/89

**Comment by miguelriosoliveira** [comment]
Please accept this PR 🙏
The warnings that hellosign-sdk brings to the project are really annoying 😪

**Review by miguelriosoliveira — DISMISSED** [review]

**Comment by dannypaz** [comment]
@nathanbuchar can we get a review/merge for this?

**Review by miguelriosoliveira — APPROVED** [review]

**Comment by charakajg** [comment]
@jtreminio-dropbox This PR requires an approval from a reviewer with write access. Hope we can merge this, although it's deprecated.

**Comment by jtreminio-dropbox** [comment]
Please use the new OpenAPI SDK: https://www.npmjs.com/package/@dropbox/sign

**Pull request closed without merging**
